float64 P
float64 I
float64 D
---
bool result
